sources/co/vac/cartago.json

```diff
@@ -0,0 +1,33 @@
+{
+    "coverage": {
+        "ISO 3166": {
+            "alpha2": "CO-VAC",
+            "country": "Cartago",
+            "region": "Valle del Cauca"
+        },
+        "country": "co",
+        "state": "vac"
+    },
+    "schema": 2,
+    "layers": {
+        "addresses": [
+            {
+                "name": "city",
+                "data": "https://addressforall-my.sharepoint.com/personal/operacao_addressforall_org/_layouts/15/download.aspx?share=EdyPQRg2TShKgpnV5DD2Y0oBZbdPpsjUcI0qVczflqdwOQ",
+                "website": "https://github.com/digital-guard/preserv-CO/tree/main/data/_pk0006.02",
+                "protocol": "http",
+                "license": {
+                    "url": "https://creativecommons.org/licenses/by/4.0/",
+                    "attribution": true,
+                    "attribution name": "CC-BY-4.0",
```

Comment:
We need to update this with the correct attribution information, not the license name.